themes/openy_themes/openy_rose/templates/paragraph/paragraph--gallery--default.html.twig

```diff
@@ -0,0 +1,92 @@
+ * - content: All paragraph items. Use {{ content }} to print them all,
+ *   or print a subset such as {{ content.field_example }}. Use
+ *   {{ content|without('field_example') }} to temporarily suppress the printing
+ *   of a given child element.
+ * - attributes: HTML attributes for the containing element.
+ *   The attributes.class element may contain one or more of the following
+ *   classes:
+ *   - paragraphs: The current template type (also known as a "theming hook").
+ *   - paragraphs--type-[type]: The current paragraphs type. For example, if the paragraph is an
+ *     "Image" it would result in "paragraphs--type--image". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - paragraphs--view-mode--[view_mode]: The View Mode of the paragraph; for example, a
+ *     preview would result in: "paragraphs--view-mode--preview", and
+ *     default: "paragraphs--view-mode--default".
+ * - view_mode: View mode; for example, "preview" or "full".
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_paragraph()
+ *
+ * @ingroup themeable
+ */
+#}
+{%
+set classes = [
+  'paragraph',
+  'paragraph-gallery',
+  'paragraph--type--' ~ paragraph.bundle|clean_class,
+  view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
+]
+%}
+{{ kint(content.field_prgf_images) }}
+{% if content.field_prgf_images['#items']|length >= 1 %}
+<div{{ attributes.addClass(classes) }}>
+  <div id="carousel" class="carousel slide" data-ride="carousel">
+    <div class="carousel-inner" role="listbox">
+      {% for key, value in content.field_prgf_images['#items'] %}
+      <div class="item{% if key == 0 %} active{% endif %}">
+        {{ content.field_prgf_images[key] }}
+        {% if content.field_prgf_images[key]['#media'].field_media_caption.0.value is not empty %}
+          <div class="container item__caption__wrapper hidden-xs">
+            <div class="row">
+              <div class="item__caption">
+                {{ content.field_prgf_images[key]['#media'].field_media_caption.0.value }}
+              </div>
+            </div>
+          </div>
+        {% endif %}
+      </div>
+      {% endfor %}
+    </div>
+    {% if content.field_prgf_images['#items']|length > 1 %}
+    <a class="left carousel-control" href="#carousel" role="button" data-slide="prev">
+      <span class="fa fa-chevron-left" aria-hidden="true"></span>
+      <span class="sr-only">{{ 'Previous'|t }}</span>
+    </a>
+    <a class="right carousel-control" href="#carousel" role="button" data-slide="next">
+      <span class="fa fa-chevron-right" aria-hidden="true"></span>
+      <span class="sr-only">{{ 'Next'|t }}</span>
+    </a>
+    {% endif %}
+  </div>
+  {% if content.field_prgf_headline['#field_name'] is not empty %}
```

Comment:
Use set instruction with ternary operators (at the top of the template) to get values from the fields with default fallback
Using direct data in if prohibited by best practices (without defaults debugging in a future is a bit tough)
Example
```twig
{% set color = content.field_blog_color ? content.field_blog_color : 'red' %}
```